.github/workflows/schedule.yml

```
@@ -9,22 +9,18 @@ jobs:
   send:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up JDK 17
-        uses: actions/setup-java@v4
+      - name: Login to Container Registry
+        uses: docker/login-action@v2
```

actionlint (1.7.8)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)